src/org/javaday/java8/misused/optional/HundredAndOneApproach.java

```diff
@@ -0,0 +1,114 @@
+package org.javaday.java8.misused.optional;
+
+import org.javaday.java8.misused.Annotations.Bad;
+import org.javaday.java8.misused.Annotations.Good;
+import org.javaday.java8.misused.Annotations.Ugly;
+
+import java.util.Optional;
+
+import static java.util.Optional.empty;
+import static java.util.Optional.ofNullable;
+
+public class HundredAndOneApproach {
+
```

Comment:
As for me new line separator between class declaration and method declaration improve readability. Cause ultimately we want to have a method in the spotlight. You disagree?